modules/project/service-agents.tf

```diff
@@ -142,16 +141,6 @@ data "google_bigquery_default_service_account" "bq_sa" {
   depends_on = [google_project_service.project_services]
 }
 
-moved {
-  from = google_project_service_identity.jit_si
-  to   = google_project_service_identity.default
-}
-
-moved {
-  from = google_project_service_identity.servicenetworking[0]
-  to   = google_project_service_identity.default["servicenetworking.googleapis.com"]
-}
-
```

Comment:
I'd keep the move blocks

Reply:
They are pretty old, you sure we want to keep them? They prevent defining higher level moved blocks on the same resources (eg when migrating FAST)

Reply:
Yes. IMO all moved blocks should be kept indefinitely. 

But your call if they're causing you pain

Reply:
No biggie, I tend to remove them after a few releases, as I had issues in the past. Let's keep these a bit more.